src/components/RelatedDocumentsList/RelatedDocumentsListPresentation.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router'
+
+// FPCC
+import getIcon from 'common/utils/getIcon'
+
+function RelatedDocumentsListPresentation({
+  documents,
+  labelStyling = 'text-left font-medium text-lg uppercase text-charcoal-900',
+}) {
+  return (
+    documents?.length > 0 && (
+      <ul className="w-full">
+        <h4 className={`${labelStyling} mb-2`}>Related Documents</h4>
```

Comment:
This header and div needs to go outside the list element in order to be valid HTML. https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/ul